stablehlo/dialect/Version.cpp

```diff
@@ -85,7 +85,7 @@ Version Version::fromCompatibilityRequirement(
     case CompatibilityRequirement::WEEK_4:
       return Version(1, 7, 0);  // v1.7.0 - Sept 05, 2024
     case CompatibilityRequirement::WEEK_12:
-      return Version(1, 1, 0);  // v1.1.0 - May 30, 2024
+      return Version(1, 3, 0);  // v1.3.0 - Jul 10, 2024
```

Comment:
Per the comment above, we use the date a release was tagged, which was July 16th:

<img width="370" alt="image" src="https://github.com/user-attachments/assets/4b225f39-2d7b-4033-b4cf-e686c5103602">

Still 12w ago (actually 12w perfectly 🎉 ) but could you update the comment to use the tag date?

Reply:
fixed